src/CoreWCF.Templates/src/templates/CoreWCFService/IService.cs

```diff
@@ -0,0 +1,61 @@
+﻿#if(!impliciteUsings)
+using System.Runtime.Serialization;
+using CoreWCF;
+using System;
+#endif
+
+namespace CoreWCFService
+{
+    [ServiceContract]
+    public interface IService
+    {
+        [OperationContract]
+        string GetData(int value);
+
+        [OperationContract]
+        CompositeType GetDataUsingDataContract(CompositeType composite);
+    }
+
+    public class Service : IService
+    {
+        public string GetData(int value)
+        {
+            return string.Format("You entered: {0}", value);
+        }
+
+        public CompositeType GetDataUsingDataContract(CompositeType composite)
+        {
+            if (composite == null)
+            {
+                throw new ArgumentNullException("composite");
+            }
+            if (composite.BoolValue)
+            {
+                composite.StringValue += "Suffix";
+            }
+            return composite;
+        }
+    }
+
+    // Utilisez un contrat de données comme indiqué dans l'exemple ci-après pour ajouter les types composites aux opérations de service.
```

Comment:
Can we have the comment in English please - I don't think we have localization support for templates. I'm assuming this is the french version of "// Use a data contract as illustrated in the sample below to add composite types to service operations."

Reply:
Sure

Reply:
I took it from the .NET Framework project, so I'm assuming its the same.